forch/faucetizer.py

```diff
@@ -122,9 +122,9 @@ def process_device_behavior(self, eth_src, behavior, static=False):
 
             self.flush_behavioral_config()
 
-    def validate_tail_acl_config(self, tail_acl):
+    def validate_tail_acl_config(self):
```

Comment:
validate is a verb but this function doesn't do anything.  tail_acl_config_valid()?

Reply:
right.. Done